python/TestHarness/RunParallel.py

```diff
@@ -0,0 +1 @@
+schedulers/RunParallel.py
```

Comment:
Create symbolic link to new RunParallel. This is because we
moved RunParallel but we continue to use 'pathtool' which
places all python modules on a flat namespace, and the old
RunParallel's pyc file will be loaded instead of the new one.